code_samples/sql-data-types.js

```diff
@@ -54,15 +54,14 @@ const varcharExample = async (client) => {
                 'valueFormat' = 'varchar'
             )
         `;
-    // executions are async, await on update count to wait for execution.
-    await client.getSql().execute(createMappingQuery).getUpdateCount();
+    await client.getSql().execute(createMappingQuery);
 
     for (let key = 0; key < 10; key++) {
         await someMap.set(key, key.toString());
     }
 
     try {
-        const result = client.getSql().execute('SELECT * FROM varcharMap WHERE this = ? OR this = ?', ['7', '2']);
+        const result = await client.getSql().execute('SELECT * FROM varcharMap WHERE this = ? OR this = ?', ['7', '2']);
         const rowMetadata = await result.getRowMetadata();
```

Comment:
We don't need to await the result.xxx methods (apart from close) anymore. Can you update those as well?